docs/patterns/single-new-eks-observability-accelerators/single-new-eks-awsnative-fargate-observability.md

```diff
@@ -0,0 +1,164 @@
+# Single New EKS Fargate Cluster AWS Native Observability Accelerator
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Single EKS Fargate Cluster Native Observability pattern using AWS native tools such as CloudWatch Logs and Container Insights.
+
+![Architecture](../images/fargate-container-insights.png)
```

Comment:
Architecture is dropped in the wrong foler it should be in patterns/images. Also the architecture you picked from the reference i shared is wrong. If you read that best practice guide fully what works with Fargate is Prometheus receiver in ADOT pipeline which is what ADOT addon is doing.